shinysdr/test/test_telemetry.py

```diff
@@ -36,9 +36,24 @@ def test_init_from_partial_json(self):
             }))
 
 
+class TestClock(Clock):
```

Comment:
Move this below the test that uses it. (In general, order non-constants from high-level/exported to implementation details unless there's an actual ordering dependency requiring otherwise.)

Rename it so it does not start with "Test" to make it clear this is not a `TestCase` subclass. (I'd be inclined to express my annoyance by calling it `SlightlyBetterClock`, absent a better idea.)